app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def index
+    greeting = Greeting.all.sample.message
+    render json: { greeting: }, status: 200
+  end
+end
```

Comment:
# Please Display the random greeting in your Greeting component.
- Both Server is well connected. Make sure that the greetings are displayed. 

  
![WhatsApp Image 2022-09-13 at 10 10 42 PM](https://user-images.githubusercontent.com/63972670/190064888-18f6018c-3665-40d5-a9ec-e2161da082ca.jpeg)

Reply:
Hello @OLIPLICHE ! thanks for the review, but i have a quiestion why you have the const url in the reducer without the '/greet' thats the root where i call to the GET the random message (see my capture from the front end repo)
![image](https://user-images.githubusercontent.com/58194121/190152673-2261beea-bfbc-4053-bba9-96c644a1a9ef.png)

if you open the react server on your browser you gonna see something like this:
![image](https://user-images.githubusercontent.com/58194121/190154845-2839ae16-b182-4fc4-9b30-df681d719d8c.png)
as you can see in the index i dont have the call to the api is in the click on the link and that sent you to the '/greet' and here is where the recuder make the call action GET to the server
![image](https://user-images.githubusercontent.com/58194121/190155750-c222db5c-329a-4120-849d-3bd02977c541.png)
![image](https://user-images.githubusercontent.com/58194121/190155953-c06279c4-1160-4e02-a29b-eba7f48bdaa3.png)

Reply:
- Hi @rega1237. Nice Job. The Url Link in the reducer is the localhost link I copied when I ran your API in local machine. "Maybe" I omitted to call the root. As I mentioned earlier, your servers are well connected without an error. Great Job for raising this. You can submit it. The next reviewer gonna review it.